packages/cli/test/integration.test.ts

```diff
@@ -25,17 +24,51 @@ import pkg from '../package.json';
 import prepareFixtures from './helpers/prepare';
 import { fetchTokenWithRetry } from '../../../test/lib/deployment/now-deploy';
 import { once } from 'node:events';
+import type http from 'http';
```

Comment:
```suggestion
import type { IncomingMessage, ServerResponse } from 'http';
```

Nit pick

Reply:
I like it with the namespace because it was a bit confusing which response-like type I was dealing with at times. I wouldn't mind it either way, though.

Reply:
Imagine someone tries this:

```ts
http.createServer(handler)
```

It will fail. So then they right click to import and that adds

```ts
import http from 'http'
```

I think that would conflict where the explicit import wouldn't.

Also I assume explicit is faster for TS but not sure 🤔

Reply:
In my VS Code (and whatever extensions/settings I have), I see this error:

<img width="790" alt="Screenshot 2023-03-17 at 1 44 17 AM" src="https://user-images.githubusercontent.com/41545/225832208-7e2d671d-0253-441f-a7ca-a6cc8e9f727b.png">

and there was no auto-import or option to add/change the import.

If I remove the `type` from the import line, everything works. This seems like a reasonable situation to me.

Do other IDEs act differently here?

Reply:
Strange, mine doesn't give me an error when importing the type but using as a runtime module 🤔

<img width="847" alt="image" src="https://user-images.githubusercontent.com/229881/225931216-ed995a7e-3751-4408-9c10-60ef38d59d4d.png">

Reply:
Oh its because of the `// @ts-nocheck` at the top (I didn't checkout this branch 🤦 )